yt/geometry/coordinates/cartesian_coordinates.py

```diff
@@ -534,10 +534,10 @@ def _ortho_pixelize(
         else:
             pixelize_cartesian(
                 buff,
-                data_source["px"],
-                data_source["py"],
-                data_source["pdx"],
-                data_source["pdy"],
+                data_source[("all", "px")],
+                data_source[("all", "py")],
+                data_source[("all", "pdx")],
+                data_source[("all", "pdy")],
```

Comment:
Really unsure about these ones. I'd need some input from @matthewturk  on how these `px` and `pdx` thinggies work.

Note that my doubts extend in the entire file.